### PR TITLE
Change/1583 various copy changes

### PR DESCRIPTION
**Pull request by libuk**
# Context
Various copy changes.

# Changes in this PR
* Update eligibility reasons copy
* Update offending summary copy
* Update copy when a referral is ready for submission
* Standardise copy when referring to CAS2

### Before
Screenshot 2023-10-10 at 09 03 51
Screenshot 2023-10-10 at 09 24 48
Screenshot 2023-10-09 at 16 26 20
Screenshot 2023-10-10 at 09 31 24

### After
Screenshot 2023-10-10 at 09 03 28
Screenshot 2023-10-10 at 09 24 54
Screenshot 2023-10-09 at 16 26 49
Screenshot 2023-10-10 at 09 31 05

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

**Comment by libuk** [comment]
[Copy changes](https://trello.com/c/O3ScY0Bc/1583-copy-changes)

File: server/form-pages/apply/assess-placement-risks-and-needs/behaviour-in-cas/previousStaysDetails.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const accommodationTypes = {
     text: 'Approved Premises (AP or CAS1)',
   },
   cas2: {
-    text: 'Bail Accommodation and Support Service (BASS or CAS2)',
+    text: 'CAS2 (formerly Bail Accommodation Support Services)',
```
Comment (tahb) [inline server/form-pages/apply/assess-placement-risks-and-needs/behaviour-in-cas/previousStaysDetails.ts:13]: For my understanding: if we wanted to define the name of CAS2 singularly, do you know how we could do that?

In Rails I could use [the internationalisation API](https://guides.rubyonrails.org/i18n.html#the-public-i18n-api) as a way to give me a variable where we could say `I18n.t('service.identifier.cas2')`

File: server/form-pages/apply/accommodation-need/sentence-information/offendingSummary.test.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ describe('OffendingSummary', () => {
     it('sets the body', () => {
       const page = new OffendingSummary(body, application)
 
-      expect(page.title).toEqual("Provide a brief summary of John Smith's offending history")
+      expect(page.title).toEqual("Provide a brief summary of John Smith's index offence(s) and offending history")
```
Comment (tahb) [inline server/form-pages/apply/accommodation-need/sentence-information/offendingSummary.test.ts:26]: Lingering fixup! commit 👍

File: server/views/applications/show.njk [hunk 3]
```diff
@@ -26,7 +26,12 @@
 
       {{ showErrorSummary(errorSummary) }}
 
-      <h2 class="govuk-heading-s govuk-!-margin-bottom-2" data-cy-status>Referral {{ taskList.status }}</h2>
+      {% if taskList.status === 'complete' %}
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-2" data-cy-status>Submit your referral below</h2>
```
Comment (tahb) [inline server/views/applications/show.njk:30]: Do we need a test for this bit?

**Review by tahb — APPROVED** [review]
LGTM once that commit is fixed up 👍

File: server/form-pages/apply/assess-placement-risks-and-needs/behaviour-in-cas/previousStaysDetails.ts (hunk 1 above)
Comment (libuk) [reply server/form-pages/apply/assess-placement-risks-and-needs/behaviour-in-cas/previousStaysDetails.ts:13]: We could use an enum:

```ts
enum ServiceNames {
  CAS2 = "CAS2 (formerly Bail Accommodation Support Services)",
  CAS3 = "Transitional Accommodation (CAS3)"
}

const text = `Service name is ${ServiceNames.CAS2}`
```

Now that we've standardised Cas2 and Cas3, it'd be worth going through and refactoring. Shall I create a card?

File: server/views/applications/show.njk (hunk 3 above)
Comment (libuk) [reply server/views/applications/show.njk:30]: I'd opt to not on this occasion, as it's quite low risk.

**Pull request merged**